modules/nf-core/nanofilt/main.nf

```diff
@@ -0,0 +1,51 @@
+process NANOFILT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/nanofilt:2.8.0--py_0':
+        'biocontainers/nanofilt:2.8.0--py_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    val   readlength
+    val   readqual
+
+    output:
+    tuple val(meta), path("*.fastq.gz"), emit: filtreads
```

Comment:
There's an optional log file `--logfile LOGFILE` that should be included as optional output too

Reply:
I added this output with pattern `*.log` and specified in the meta.yml file that this should be the extension if using this option through `ext.args`